### PR TITLE
[Analytics] Implement analytics navigation

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)
- Responsive top navbar and a sidebar.
- Collapsible airqlouds dropdown.
- Reconfigured absolute imports to allow imports from external folders like public.
- Keyboard accessibility for users unable to use a mouse.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* git pull origin staging
* git checkout ft-aq-analytics-nav
* yarn install
* yarn add
* Open http://localhost:3000 . The Analytics dropdown item in the sidebar is active. You can toggle it and select the airqlouds option to navigate to the airqlouds page or follow this link http://localhost:3000/analytics/airqlouds

#### What are the relevant tickets?
- [AN-137](https://airqoteam.atlassian.net/browse/AN-137)

#### Screenshots
aq_nav_structure_3
aq_nav_structure_2
aq_nav_structure_1

### CODE REVIEW

**Review by 123MwanjeMike — CHANGES_REQUESTED** [review]
image
Great progress @Codebmk 

**I however notice the tests are failing cause of an import configuration. Could you look into it?**
image

**Review by BenjaminSsempala — APPROVED** [review]
LGTM

**Review by 123MwanjeMike — APPROVED** [review]
Great work @Codebmk, the tests workflow is now passing.
LGTM

File: platform/.eslintrc.json [hunk 1]
```diff
@@ -8,8 +8,12 @@
...
     "import/resolver": {
       "node": {
-        "paths": ["src"],
-        "extensions": [".js", ".jsx"]
+        "paths": ["."],
+        "extensions": [".js", ".jsx"],
+        "moduleDirectory": ["src", "node_modules"]
+      },
+      "jsconfig": {
+        "config": "jsconfig.json"
```
Comment (Codebmk) [inline platform/.eslintrc.json:16]: This imports the aliases from the paths in the jsconfig.json

**Review by FaithDaka — APPROVED** [review]

**Pull request merged**